14-draft.json

```diff
@@ -1135,7 +1135,7 @@
       }
     },
     "membership_plans": {
-        "description": "A list of the different membership plans your hackerspace might have.",
+        "description": "A list of the different membership plans your hackerspace might have. Try to use the minimal amount of information needed to describe the membership plan. For example, if your membership fee is 10€ per month, but you bill it yearly (aka. the member pays the fee once per year), set the amount to 120 an the renewal_interval to yearly.",
```

Comment:
I'm not yet convinced of the second sentence. What about changing it to something like "Set the value according to your billing process."?